### DISCUSSION TITLE
Texture color does not represent image color

### DISCUSSION THREAD

**Discussion by AlexanderJuestel** [pyvista/pyvista#4040] [Q&A]
Hello, 
I am trying to apply a texture (a geological map) to a digital elevation model. The application itself works fine. The only issue is that the colors are wrong. The first picture shows the map in PyVista with the wrong colors. The second one with the correct colors. The image was loaded using PIL. The texture is created below and will be applied during the plotting. The structured grid is created from a numpy meshgrid. I attached both files for reference. 

```ruby
from PIL import Image

im = Image.open('GK_Map.png') # Can be many different formats.
# Getting pixel values
pixel_values = list(im.getdata())
# Reshaping pixel values to match dimensions of DEM
array = np.array(pixel_values).reshape(1027, 1381,3)

# Creating StructuredGrid
mesh = pv.StructuredGrid(x, y, np.flipud(elevation))
mesh.texture_map_to_plane(inplace=True)

# Converting array with pixel values to texture
texture = pv.numpy_to_texture(array)
```

![image](https://user-images.githubusercontent.com/45469915/219849166-942742e6-eb52-40a0-b0fd-5286c177e1ce.png)

![image](https://user-images.githubusercontent.com/45469915/219849220-711e69f1-2488-4d83-a0c9-c5be938bb87d.png)

![GK_Map](https://user-images.githubusercontent.com/45469915/219849350-5b1ec9cf-0c09-464b-a9a7-c1bd031ee359.png)
[DEM10_EPSG_32632_WPT_c1_cropped.zip](https://github.com/pyvista/pyvista/files/10773665/DEM10_EPSG_32632_WPT_c1_cropped.zip)

**Comment by banesullivan** [comment]
Well, this is quite odd. Would you be able to make this example fully reproducible for me? I see a tiff attached but not `GK_Map.png`. Further all the necessary code to load everything

**Reply by AlexanderJuestel** [reply]
Yep, will do that first thing tomorrow morning :)

**Comment by AlexanderJuestel** [comment]
@banesullivan, yes, it is odd. I created the reproducible example for you and it suddenly works :D 

![image](https://user-images.githubusercontent.com/45469915/220037759-16cf899c-19ab-4f55-a83f-0150aeea2bcf.png)

**Reply by banesullivan** [reply]
That makes me think something is going awry with the numpy array of the texture you were originally loading
